errors/src/parser/parser_errors.rs

```diff
@@ -223,4 +223,12 @@ create_errors!(
         msg: "Array dimensions specified as a tuple cannot be empty.",
         help: None,
     }
+
+    /// For when the parser encountered a deprecated `circuit {...}` structure.
+    @formatted
+    cricuit_definition {
+        args: (),
+        msg: "\"circuit {...})\" is deprecated. Did you mean struct?",
```

Comment:
It's not deprecated if its an error; it's no longer accepted. Deprecated suggests a warning instead.

Reply:
Ah, that's what I was initially going for as to not fully remove it in a single update. I believe throwing an error instead of a warning it breaks some of the tests. Especially where testing the CLI is concerned cause it would pull packages that use the old circuit syntax. As well as any other packages currently on the APM.

Reply:
Ah wait I see what you mean.